.github/workflows/build.yml

```diff
@@ -13,6 +14,7 @@ on:
     # Must stay in sync with the paths in .github/workflows/docs.yml and .github/mergify.yml
     paths-ignore:
       - '**/*.md'
+      - '**/*.png'
```

Comment:
Can you update `docs.yml` and `mergify.yml` as suggested by the comment a few lines above?

Reply:
@russellb correct me if Im wrong here, but those look like they are for triggering markdown-lint in doc.yml or checking for a md-lint pass in mergify?

Reply:
Yes, that's the purpose of the files.

Here's the problem with the change as-is. Imagine a PR that only changes a png file.
- The build workflow does not run as a result of this change.
- Mergify will never merge the PR because the rules based on file types says the build workflow must pass.

An alternative is to make sure that the docs workflow always runs for png file changes. That requires changing docs.yml and then changing mergify.yml to have it look for the docs job passing.

I'll push a patch for you.

Reply:
@nerdalert take a look at the commit I added to the branch. I wanted you to see what I meant before it merges. After you take a look, you can drop the `hold` label so this will merge.

Reply:
I dropped the hold, just take a look whenever :)